src/layouts/components/header.layout.tsx

```
@@ -0,0 +1,141 @@
+                                <NotificationsActiveRounded color="action" />
+                            </Badge>
+                        </IconButton>
+                        <DropdownMenu
+                            user={user}
+                            title={user.name}
+                            aria-label={user.name}
+                        />
+                    </Stack>
+                </Toolbar>
+            </AppBar>
+            <div style={{ marginBottom: '8rem' }}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)